Frameworks/UIKit/NSString+UIKitAdditions.mm

```diff
@@ -41,6 +41,18 @@ void NSStringForceinclude() {
 
 @implementation NSString (UIKitAdditions)
 
+static CTTextAlignment __UITextAlignmentToCTTextAlignment(UITextAlignment alignment) {
```

Comment:
eesh

---
In reply to: [85569934](https://github.com/Microsoft/WinObjC/pull/1254#discussion_r85569934) [](ancestors = 85569934)